pkg/cloud/onprem/onprem.go

```diff
@@ -56,5 +59,23 @@ func NewOnPremBuiltin(kubeConfigFile string) *starlark.Builtin {
 
 // KubeConfig is part of the cloud.KubernetesVendor interface.
 func (o *OnPrem) KubeConfig(ctx context.Context) (*rest.Config, error) {
+	kubeConfigVaultPath := o.AbstractKubeVendor.AddonSkyCtx(
+		map[string]string{}).Attrs["vaultkubeconfig"].(starlark.String).String()
```

Comment:
This line will fail if `vaultkubeconfig` isn't set, however the next block allows for the case that `vaultkubeconfig` wasn't set. I addressed it in #27

Reply:
Nice catch, and thank you for the PR fix.